### ISSUE TITLE
"From Coq Require Import XYZ" error message refers to Stdlib

### ISSUE THREAD

**Issue by jfehrle** [coq/coq#19606]
### Description of the problem

Would be better if the error message referred to what the user entered, i.e. "Coq" instead of "Stdlib", if easy to fix.  It could be confusing.  Or at least show the warning `"From Coq" has been replaced by "From Stdlib"` in addition to the error message.

EDIT: Using coqc, I can see that the warning is generated but it's not displayed in CoqIDE.

![image](https://github.com/user-attachments/assets/8da0bb32-cc2b-4264-84ae-a6e1b4a61637)

### Small Coq file to reproduce the bug

```coq
From Coq Require Import ABC.
```

### Version of Coq where this bug occurs

8.21+alpha

**Comment by proux01** [comment]
No, the actual require is indeed Stdlib so the message should be about Stdlib. Before the error, you get a warning explaining that Coq has been translated to Stdlib. (then maybe the interface doesn't display it properly, but that's an interface bug, not a coq bug per se)

**Comment by Zimmi48** [comment]
This had already been triaged as a usability issue of CoqIDE, not a Coq bug.

**Comment by proux01** [comment]
My bad, I missed it.

**Comment by ejgallego** [comment]
I confirm that this example displays both the error and the warning in coq-lsp.

Tho indeed, the whole setup is IMHO a bit confusing usability wise.

**Comment by ejgallego** [comment]
On the other hand the quickfix codeaction works, so IMHO that makes the situation much better.
